server/fuzzmanager_setup_helper.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
```

Comment:
Thanks for contributing this helper script :) Could you maybe make a new directory server/contrib/ and move the script there? Then move the script output from README.md to a new README.md in server/contrib/ so we don't clutter the main README too much. I will add a reference to your script in main README.md myself and also edit the contrib README later on so we can add more user-contributed setup helpers. Thanks again!